device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D100C12S2/sai.profile

```diff
@@ -0,0 +1,2 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_3800_2x10g_28x50g_48x100g.xml
```

Comment:
why is the xml file different from` sai_3800_2x10g_100x50g_12x100g.xml`?

Reply:
How is this change tested?

Reply:
okay, this has to be fixed. I will fix it.